src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+            </a>{' '}
+            which need more support to grow their impact and improve quality of
+            life for people all over the developing world.
+          </p>
+
+          <p>
+            Another interesting approach is directly capturing carbon dioxide
+            from the air is another interesting approach which seems to be
+            slowly improving over time, with
+            <a href="">Climeworks</a> https://climeworks.shop/) offering a
+            monthly subscription to back their project, which is about €50 a
+            month to remove 0.6 tonnes from the environment, which over could
+            wipe out 7.2 tonnes for €800. Are you willing to pay €800 a month to
+            offset most of your footprint?
+          </p>
+
+          <p>
+            Sending money to projects like this is solving problems around the
+            world, reducing and trapping emissions, and hopefully if they all
+            scale enough through any means then we'll see emissions drop, but
+            most people cannot afford to offset the amount they actually
+            produce: because we all produce too much CO2e and offsetting it
+            fully and legitimately is expensive.{' '}
+          </p>
+
+          <h2 id="reducingvsoffsetting">Reducing vs Offsetting</h2>
+
+          <p>
+            Most of the problems folks have with offsetting stems from some
+            companies (usually the greenwashers) pretending that we can just pay
+            a little money, continue driving, continue flying, and continue
+            eating an overly meat-rich diet, and we completely agree there.
+          </p>
+
+          <p>
+            There's only so many rocket stoves, wind farms, and biodigesters we
+            can give to folks before those sources of CO2 avoidance are tapped
+            out, at which point we have to look for other sources of carbon
+            emissions: like flying, driving, and what we eat. There's no point
+            in delaying this step, we need to start avoiding these emissions
+            immediately.
+          </p>
+
+          <p>
+            To avoid the worst effects of climate change, we all get our
+            individual carbon footprints down to{' '}
+            <a href="https://www.tmrow.com/climatechange#objective--2-tons-co2eq-per-human-per-year-by-2050">
+              5 tonnes a year by 2030
+            </a>
+            , then 2 tonnes a year by 2050. Whilst the previous examples of
+            United Kingdom (10 tCO2e per person per year) and United States (20
+            tCO2e per person per year), the{' '}
+            <a href="https://www.independent.co.uk/environment/british-carbon-footprint-africa-emissions-oxfam-climate-change-a9271861.html">
+              18 lowest emitting countries were in Africa, including Nigeria,
+              Kenya and Zimbabwe
+            </a>
+            , all coming in under 1 tCO2e per person per year! it's worth
```

Comment:
This sentence is missing something.  It reads:
"Whilst the... uk, us, the 18 lowest emitting countries were in africa, all coming in..'  Add something... 'Whilst the UK and US **are the highest**, the lowest ....